softgroup/basis/common-router/src/test/java/com/softgroup/common/router/api/AbstractRequestHandlerTest.java

```diff
@@ -0,0 +1,30 @@
+package com.softgroup.common.router.api;
+
+import com.softgroup.authorization.api.message.LoginRequestData;
+import com.softgroup.authorization.api.message.LoginResponseData;
+import com.softgroup.common.protocol.Request;
+import com.softgroup.common.protocol.Response;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class AbstractRequestHandlerTest {
+    @Test
+    public void test() {
+        AbstractRequestHandler<LoginRequestData, LoginResponseData> arh = mock(AbstractRequestHandler.class, Mockito.CALLS_REAL_METHODS);
+
+        Request<LoginRequestData> loginRequest = mock(Request.class);
+        Response<LoginResponseData> loginResponse = mock(Response.class);
```

Comment:
@VasjaVn Use real objects! not mock. Try to think what your handle method must do. This test is not testing work-flow of that method.